corehq/form_processor/abstract_models.py

```diff
@@ -155,6 +155,65 @@ def get_subcases(self):
     def get_case_property(self, property):
         raise NotImplementedError
 
+    def get_json(self, lite=False):
```

Comment:
we should use our serializers for this if possible: https://github.com/dimagi/commcare-hq/blob/master/corehq/form_processor/serializers.py